Makefile

```diff
@@ -73,7 +73,7 @@ yamldocs: ## generate documentation YAML files consumed by docs repo
 
 .PHONY: shellcheck
 shellcheck: ## run shellcheck validation
-	scripts/validate/shellcheck
+	docker buildx bake shellcheck
```

Comment:
Similar (although `scripts/validate/shellcheck` could still work if you have it installed locally); this one should probably go in `docker.Makefile`